ckanharvester.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+from sqlalchemy import create_engine
+from sqlalchemy import MetaData
+from ckanext.harvest.model import HarvestObject
+import logging
+from base import HarvesterBase
+
+log = logging.getLogger(__name__)
+
+
+class CKANHarvester(HarvesterBase):
+    def _connect(self, user, password, db, host='192.168.56.101'):
+        url = 'mysql://{}:{}@{}/{}?charset=utf8'
```

Comment:
I think, this whole connection url should go to the configuration file as it [was before](https://github.com/ivpk/opendata.gov.lt-mysql-import/blob/master/odgovltimport.py#L397):

```python
 engine = sa.create_engine(config['odgovltimport.externaldb.url'])
```

And it would be nice if you keep the same configuration name, because it is already included in the deployment scripts:

https://github.com/ivpk/opendata.gov.lt/blob/master/roles/ckan/templates/ckan.ini.j2#L193